ethereum/contracts/BasicInboundChannelV2.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.5;
+pragma experimental ABIEncoderV2;
+
+import "./ParachainLightClient.sol";
+import "./BeefyLightClient.sol";
+import "./SimplifiedMMRVerification.sol";
+
+contract BasicInboundChannel {
```

Comment:
Should be named BasicInboundChannelV2 so as not to clash with the previous version.

Reply:
or rather MultiUserChannel